routers/graphql-router.js

```diff
@@ -15,7 +15,9 @@ dbConnector.connect().then(() => {
     const userCollection = new MongoDBCollection(dbConnector.client, DATABASE_NAME, USER_COLLECTION);
     const dataInterface = new User(userCollection);
     root = {
-        getMyUser : dataInterface.getMyUser.bind(dataInterface)
+        version: () => {return config.version},
```

Comment:
this version graphql call is not in the schema so it can be removed because it cannot be called